paasta_tools/marathon_tools.py

```diff
@@ -657,17 +658,27 @@ def format_marathon_app_dict(self) -> FormattedMarathonAppDict:
         docker_volumes = self.get_volumes(system_volumes=system_paasta_config.get_volumes())
 
         net = get_mesos_network_for_net(self.get_net())
-
-        complete_config: FormattedMarathonAppDict = {
-            'container': {
+        container_dict: MarathonContainerInfo = {
+            'docker': {
+                'image': docker_url,
+                'network': net,
+                "parameters": self.format_docker_parameters(),
+            },
+            'type': 'DOCKER',
+            'volumes': docker_volumes,
+        }
+        # Only image and forcePullImage are supported: "Mesos Containerizer and Universal Container Runtime"
+        # in http://mesosphere.github.io/marathon/1.4/docs/native-docker.html
+        if self.get_containerizer_type() == 'MESOS':
+            container_dict = {  # type: ignore
```

Comment:
Can you change this to do a normal if/then with the container_type instead of mutating `container_dict` ?